### PR TITLE
OCPBUGS-36373: Update extra manifest examples in IBU docs

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.16, 4.17

Issue: https://issues.redhat.com/browse/OCPBUGS-36373

Link to docs preview: https://78299--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.html#cnf-image-based-upgrade-creating-backup-extra-manifests_nongitops

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mpmaruthu** [comment]
Telco RAN QE: Looks good to me.
/Peri M

**Comment by dfitzmau** [comment]
/remove-label peer-review-in-progress

/label peer-review-done

My suggestions really depend on the style adopted for YAML files in the Telco docs.

File: modules/cnf-image-based-upgrade-prep-extramanifests.adoc [hunk 1]
```diff
@@ -10,14 +10,15 @@ You can create additional manifests that you want to apply to the target cluster
...
+. Create a YAML file that contains your extra manifests, for example, for SR-IOV:
 +
+.Example SR-IOV resources
 [source,yaml]
 ----
 apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetworkNodePolicy
 metadata:
-  name: "pci-sriov-net-e5l"
+  name: "example-sriov-node-policy"
```
[the remark this answers is not in the document]
Comment (amolnar-rh) [reply modules/cnf-image-based-upgrade-prep-extramanifests.adoc:21]: We often use `"example-field-value"` without marking them as user-replaced values in the Telco docs, for example [here](https://78320--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/ztp-deploying-far-edge-sites#ztp-creating-the-site-secrets_ztp-deploying-far-edge-sites)

[the remark this answers is not in the document]
Comment (dfitzmau) [reply modules/cnf-image-based-upgrade-prep-extramanifests.adoc:21]: Thanks, @amolnar-rh . I was unsure here so thatnks for clarifying. LGTM.

**Review by aireilly — APPROVED** [review]
/lgtm

**Pull request merged**